seq2science/rules/qc.smk

```diff
@@ -116,11 +116,11 @@ if config["trimmer"] == "trimgalore":
             """
 
 
-elif config["trimmer"] == "fastp":
+elif config["trimmer"] == "fastp" or "kmer_estimation" in config:
```

Comment:
what's with this `or`? We only support these 2 trimmer right?

Reply:
Oops :see_no_evil: